pool/utils.gno

```diff
@@ -4,10 +4,17 @@ import (
 	"std"
 
 	pusers "gno.land/p/demo/users"
+
 	u256 "gno.land/p/gnoswap/uint256"
 )
 
 func a2u(addr std.Address) pusers.AddressOrName {
+	if !addr.IsValid() {
+		panic(addDetailToError(
+			errInvalidAddress,
```

Comment:
`errInvalidAddress` is not declared

Reply:
240a95632d172cfb5a08fbe0fce6f49ed32fd678